src/frontend/components/home/Home.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Home = () => {
```

Comment:
If not needed , the empty component can be deleted.

Reply:
The component will be added later, created it for redirecting after login